src/main/java/org/datadog/jenkins/plugins/datadog/model/BuildData.java

```diff
@@ -631,6 +633,7 @@ public Map<String, Set<String>> getTags() {
         return allTags;
     }
 
+    @SuppressFBWarnings
```

Comment:
It is better to suppress a specific warning rather than all of them. To suppress a specific warning, provide its name as the annotation value. The list of possible warnings to suppress can be found [here](https://github.com/archived-codacy/codacy-findbugs/tree/master/src/main/resources/docs/description).

Reply:
In all of the cases where I use `@SuppressFBWarnings`, it is to avoid calling [DCN_NULL_POINTER_EXCEPTION](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dcn-nullpointerexception-caught-dcn-nullpointer-exception), which is not in the list of the warnings that you linked. There doesn't seem to be an argument to `@SuppressWarnings` that prevents a warning for catching NPEs, and as we agreed to lean against the suggested solution of null-checking every possible NPE, I think that we should still stick to using `@SuppressFBWarnings`.

Reply:
Unfortunately it doesn't seem so--changing the annotation from `@SuppressFBWarnings` to `@SuppressFBWarnings("DCN_NULL_POINTER_EXCEPTION")` causes the DCN_NULL_POINTER_EXCEPTION to reappear. I have also tried `@SuppressWarnings("DCN_NULL_POINTER_EXCEPTION")`.

Reply:
It works for me locally. The things is the warning name is `DCN_NULLPOINTER_EXCEPTION` and not `DCN_NULL_POINTER_EXCEPTION` (mind the absence of underscore between "NULL" and "POINTER").

Reply:
So it should be `@SuppressFBWarnings("DCN_NULLPOINTER_EXCEPTION")`

Reply:
Thank you for the catch (I'm not sure how I missed that)! I am now only suppressing DCN_NULLPOINTER_EXCEPTION.